lib/ansible/modules/utilities/logic/include_role.py

```diff
@@ -19,16 +19,16 @@
 module: include_role
 short_description: Load and execute a role
 description:
-  - Loads and executes a role as a task dynamically.
-  - This frees roles from the C(roles:) directive and allows them to be treated more as tasks.
-  - Unlike M(import_role), most keywords, including loop, with_items, and conditionals, apply to this statement.
-  - The do until loop is not supported on M(include_role).
+  - Dynamically loads and executes a specified role as a task.
+  - May be used only where Ansible tasks are allowed - inside C(pre_tasks), C(tasks), or C(post_tasks) playbook objects, or as a task inside a role.
+  - Task-level keywords, L(loops,../user_guide/playbooks_loops.html), and L(conditionals,../user_guide/playbooks_conditionals.html) will be applied only to the C(include_role) statement itself. If you want the opposite behavior, use M(import_role) instead.
+  - The C(until) keyword, which retries a task until a condition is met, does not apply to C(include_role).
```

Comment:
I would rephrase since its not just `until` 'There are several keywords ignored by this action, like `until` and `retries`.